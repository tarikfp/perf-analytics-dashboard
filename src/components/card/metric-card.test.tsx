import { render } from '@testing-library/react';
import * as React from 'react';
import {
  createWithThemeContext,
  wrapWithThemeContext,
} from '../../utils/test-utils';
import MetricCard from './metric-card';

const props = {
  isLoading: false,
  title: 'metric-card',
};

test('metric card renders correctly', () => {
  const tree = createWithThemeContext(<MetricCard {...props} />);
  expect(tree).toMatchSnapshot();
});

test('metric card title text renders correctly', () => {
  const { getByText } = render(wrapWithThemeContext(<MetricCard {...props} />));
  expect(getByText(props.title)).toBeTruthy();
});
