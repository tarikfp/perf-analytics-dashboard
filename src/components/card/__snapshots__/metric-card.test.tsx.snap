// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`metric card renders correctly 1`] = `
<div
  className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-i54eqj-MuiPaper-root-MuiCard-root"
>
  <h5
    className="MuiTypography-root MuiTypography-h5 css-1673d9l-MuiTypography-root"
    style={
      Object {
        "padding": 5,
      }
    }
  >
    metric-card
  </h5>
  <hr
    className="MuiDivider-root MuiDivider-fullWidth css-9mgopn-MuiDivider-root"
  />
  <div
    className="MuiCardContent-root css-46bh2p-MuiCardContent-root"
  />
</div>
`;
